drivers/timer/arcv2_timer0.c

```diff
@@ -90,10 +90,10 @@ static u32_t announced_cycles;
  * in elapsed() function, as well as in the updates to cycle_count.
  *
  * Note:
- * Each time cycle_count is updated with the value from overflow_cyc,
- * the overflow_cyc must be reset to zero.
+ * Each time cycle_count is updated with the value from overflow_cycles,
+ * the overflow_cycles must be reset to zero.
  */
-static volatile u32_t overflow_cyc;
+static volatile u32_t overflow_cycles;
```

Comment:
Cannot we leave this variable rename out of this change? This hides important functional changes in between beautification changes.

Reply:
It's already in different commit